modules/metal-connection/README.md

```diff
@@ -0,0 +1,17 @@
+# Metal Connection SubModule
+
+The Fabric Metal Connection SubModule will create a connection from a Metal to the following Z-Side
+Access Points based on provided variable configuration:
+1. Fabric Service Profile - SP Access Point Type
+
+Please refer to the metal-* examples in this module's registry for more details on how to leverage the submodule.
+
+<!-- Begin Module Docs (Do not edit contents) -->
+## Equinix Fabric Developer Documentation
```

Comment:
You don't need any content between these boundaries. It will be overwritten anyway.